sklearn/kernel_ridge.py

```diff
@@ -94,17 +94,17 @@ class KernelRidge(MultiOutputMixin, RegressorMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
+    See Also
+    --------
+    sklearn.linear_model.Ridge : Linear ridge regression.
+    sklearn.svm.SVR : Support Vector Regression implemented using libsvm.
```

Comment:
we can avoid the detail regarding `libsvm` but we can mention that it also use accept kernels

Reply:
In this context, is it enough to replace "using libsvm" with "using kernels" ?

Reply:
I assume something like:

```
Suppor Vector Regression accepting a large variety of kernels.
```

Reply:
Thanks! I will update it right now